Sources/Settings/SettingsTabViewController.swift

```diff
@@ -153,17 +153,21 @@ final class SettingsTabViewController: NSViewController, SettingsStyleController
 		to toViewController: NSViewController,
 		options: NSViewController.TransitionOptions = [],
 		completionHandler completion: (() -> Void)? = nil
-	) {
-		let isAnimated = options
-			.isSubset(of: [
-				.crossfade,
-				.slideUp,
-				.slideDown,
-				.slideForward,
-				.slideBackward,
-				.slideLeft,
-				.slideRight
-			])
+  ) {
+    let isAnimated: Bool = {
+      if options.isEmpty {
+        return false
+      }
+      return options.isSubset(of: [
+        .crossfade,
+        .slideUp,
+        .slideDown,
+        .slideForward,
+        .slideBackward,
+        .slideLeft,
+        .slideRight
+      ])
+    }()
```

Comment:
It can just be:

```swift
let isAnimated = !options.isEmpty && options.isSubset(of: [
    .crossfade,
    .slideUp,
    .slideDown,
    .slideForward,
    .slideBackward,
    .slideLeft,
    .slideRight
])
```

Also use tab-indetation.